helm/prometheus-agent-app/templates/_helpers.tpl

```diff
@@ -32,7 +32,25 @@ application.kubernetes.io/managed-by: {{ .Release.Service | quote }}
 application.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 giantswarm.io/managed-by: {{ .Release.Name | quote }}
-giantswarm.io/service-type: {{ .Values.serviceType }}
 helm.sh/chart: {{ include "chart" . | quote }}
 {{- end -}}
 
+{{/*
+prometheus-image
+*/}}
+{{- define "prometheus-image" -}}
+{{- if eq .Values.image.tag "" -}}
```

Comment:
You can replace the whole thing using:

{{- if .Values.image.tag -}}
{{- printf "%s/%s:%s" .Values.image.registry .Values.image.name .Values.image.tag -}}
{{- else -}}
{{- printf "%s/%s:%s" .Values.image.registry .Values.image.name .Chart.AppVersion -}}
{{- end -}}

Reply:
Otherwise I'm fine